addons/website_sale/static/src/js/website_sale_payment.js

```diff
@@ -123,4 +123,22 @@ odoo.define('website_sale.payment', require => {
                 this.$submitButton.prop('disabled', disabledReasonFound);
             },
         }));
+
+    publicWidget.registry.WebsiteSalePayment = publicWidget.Widget.extend(
```

Comment:
Following the comment above, I'd move this widget to `website_sale.js` since it's not really related to `payment`.